src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -192,6 +191,21 @@ operator()(const NodeID, const EdgeID source_edge_id, Intersection intersection)
                 return intersection;
             }(intersection_node_id);
 
+            const auto link_data = node_based_graph.GetEdgeData(road.turn.eid);
+            // Check if the road continues here
+            const bool is_through_street =
+                target_intersection.end() !=
+                std::find_if(target_intersection.begin()+1,
```

Comment:
Target intersection can never be empty, correct?

Also clang-format (or is the ()+1 already formatted?).